DataTransfer/TransferFiles.py

```diff
@@ -0,0 +1,11 @@
+#python 3.6
+from github import Github
+token = "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"
+git_user = Github(token)
+repo = git_user.get_user().get_repo("TeamProjectsIIData")
+folder_prefix = "Data/"
+test_file = folder_prefix + "TestData.txt"
```

Comment:
Can you make this bottom section into a function that we can easily call? Such as "uploadFile(filename, commitMessage)". Then you would find the file in "/Data" read the contents and create the file in github.